gateway/1.4/modules/ROOT/pages/flex-gateway-k8-change-helm-settings.adoc

```diff
@@ -0,0 +1,231 @@
+= Update Pod Settings for a Flex Gateway Deployment Through a Helm Chart
+
+Change requirements for a Pod that contains a Flex Gateway deployment. The example in this guide sets the following CPU and memory requirements in the Helm chart for a Flex Gateway deployment:
+
+* `resources.limits.cpu=1500m`
+* `resources.limits.memory=2048Mi`
+* `resources.requests.cpu=260m`
+* `resources.requests.memory=384Mi`
+
+Use this example as a guide to updating any values in the Helm chart for your Flex Gateway deployment. 
+
+Find the default Helm chart settings for Flex Gateway through one of the following methods:
+
```

Comment:
A more direct way to say this:

Open...
Run...

Reply:
Updated, though I do find that the gerund emphasizes the optional nature of the methods. For similar reasons, I've always disagreed with imperative title styles like "Do X" vs. "Doing X" bc I don't need to run the steps in "Doing X" unless I have a reason to or the doc tells me to do it for some reason.